mdpow/top/oplsaa.ff/m24.itp

```diff
@@ -0,0 +1,39 @@
+; TIP3P-M24 water model from Shirts and Pande. J. Chem. Phys., 122(13), 2005. doi:10.1063/1.1877132
+; TIP3P geometry
+; epsilon: 0.24 kcal/mol
+; sigma: 3.111 Å
+
+[ atomtypes ]
```

Comment:
The `[ atomtypes ]` section is a problem. It means that this file *must* come directly after the forcefield, see #54. It would be much cleaner to 
* comment these lines 
* have new atom types like opls_111_m24 in the ff files

If any of the ligands have opls_111 then including m24 will also silently change those parameters.